SingularityExecutor/src/main/java/com/hubspot/singularity/executor/SingularityExecutorThreadChecker.java

```diff
@@ -124,6 +125,8 @@ private int getNumUsedThreads(SingularityExecutorTaskProcessCallable taskProcess
         }
       } catch (DockerException e) {
         throw new ProcessFailedException(String.format("Could not get docker root pid due to error: %s", e));
+      } catch (UncheckedTimeoutException te) {
+        throw new ProcessFailedException("Timed out trying to reach docker daemon");
```

Comment:
could we include the timeout duration here? (i.e. `Timed out trying to reach docker daemon after N seconds`)